addons/func_godot/src/map/func_godot_map.gd

```diff
@@ -82,6 +95,11 @@ func verify_and_build() -> void:
 	else:
 		emit_signal("build_failed")
 
+func _on_reimport(resources: PackedStringArray) -> void:
+	for resource in resources:
+		if resource == local_map_file:
+			verify_and_build()
```

Comment:
Should/Can we have an additional option to autobuild UV2s alongside this.
This UV2 unwrapping might not be necessary though because I think it will emit the finished building signal we can listen to

Reply:
I'll be honest, haven't touched that part of the plugin at all, so I have no idea :P
I could go read up on that part of the docs if you consider it important.

Reply:
UV2 unwrapping would have to yield to build completion, and even then the only GI that requires UV2 unwrapping is LightmapGI. You'd need an additional `auto_unwrap_uv2`, which could be handy anyway. It should _not_ be reliant on `auto_build` though, and instead be a completely separate consideration in the build process.

Reply:
It's not currently that useful since lightmap baking isn't exposed for script use currently by Godot- which is the point of the UV2 unwrapping so you have UV2s to use for the lightmap baking.

However, when it's exposed, this will be useful. This is discussed in runtime map baking, and if it's easy to add this as an option, I think it should be done. Otherwise, don't worry too much about it

Reply:
`auto_unwrap_uv2` is still useful in the sense that if you build your map and plan on using LightmapGI you can have it auto unwrap on build instead of remembering to press the button every time. You don't need to immediately bake your lightmap upon unwrapping the UV2s. I imagine it'd be useful as a sister feature to the auto rebuild, which is also not really useful at runtime either. 

Be careful that you don't tunnel vision into your runtime building use cases when judging the utility of features: you're actually in the minority of our users who use func_godot that way.